src/edu/ucsb/cs56/projects/games/subsink/Ship.java

```diff
@@ -17,7 +23,7 @@
 	 * @param y	The initial vertical position in pixels
 	 */
 	public Ship(double x, double y) {
-		super(x, y-12, 80, 15);
+		super(x, y-30, 166, 31);
```

Comment:
adjust the y-value so it's partially submerged in the water, it looks awkward right now just perching on the surface